WORKSPACE

```diff
@@ -46,6 +46,14 @@ load("@grab_bazel_common//rules:maven.bzl", "pin_bazel_common_dependencies")
 
 pin_bazel_common_dependencies()
 
+load("@rules_detekt//detekt:dependencies.bzl", "rules_detekt_dependencies")
```

Comment:
This will already be present as part of `bazel_common_setup` call no?

Reply:
yes,